src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -88,6 +88,55 @@ public void doPost_loggedOut_postForbidden() throws ServletException, IOExceptio
         .isEqualTo(0);
   }
 
+  @Test
+  public void doPost_missingLecture_badRequest() throws ServletException, IOException {
+    testServices.setEnvIsLoggedIn(true);
+
+    servlet.doPost(request, response);
+
+    verify(response).sendError(HttpServletResponse.SC_BAD_REQUEST, "Missing lecture parameter.");
```

Comment:
please create constants for the error messages

Reply:
Done!